src/custom/components/Tokens/TokensTable.tsx

```diff
@@ -43,25 +51,67 @@ export default function TokenTable({
   // token index
   const getTokenIndex = useCallback((i: number) => (page - 1) * MAX_ITEMS + i, [page])
 
+  //sorting
+  const applyDirection = useCallback((condition: boolean, sortDirection: boolean) => {
+    return condition ? (sortDirection ? -1 : 1) * -1 : (sortDirection ? -1 : 1) * 1
```

Comment:
Why the multiplication by `-1` and `1`?

I don't think it makes it any clearer

Reply:
I wrote this in a cleaner way but we still need both param's for the condition logic and multiplication.